sycl/include/CL/sycl/ONEAPI/reduction.hpp

```diff
@@ -349,10 +350,11 @@ class reduction_impl {
   using result_type = T;
   using binary_operation = BinaryOperation;
   using accessor_type =
-      accessor<T, Dims, AccMode, access::target::global_buffer, IsPlaceholder>;
+      accessor<T, Dims, AccMode, access::target::global_buffer, IsPlaceholder,
+               ONEAPI::accessor_property_list<>>;
```

Comment:
Is it really needed? The last type is already defaulted to `ONEAPI::accessor_property_list<>`.
Same question in other places with similar changes.

Reply:
It's not needed on GCC, but MSVC requires it.